index.html

```diff
@@ -410,10 +405,60 @@ <h3>JsonSchemaCredential</h3>
               }
               </pre>
             </p>
-            <p class="issue" data-number="159">
-               Add language about JsonSchemaCredential credentials having a credentialSchema property
-               and the risks of nesting.
-            </p>
+            <section class="normative">
+               <h4>jsonSchema</h4>
+               <p>
+                  This term is part of the 
+                  <a href="https://www.w3.org/2018/credentials/#JsonSchemaCredential">Verifiable Credentials Vocabulary v2.0</a>.
+               </p>
+               <p><b>jsonSchema</b> enables the use of the <code>jsonSchema</code> property
+                  within the <code>credentialSubject</code> of a verifiable credential.  The term is
+                  intended to be used with the <a href="#jsonschemacredential"></a> type. The value of
+                  the <code>jsonSchema</code> property MUST be a valid [[JSON-SCHEMA]].
```

Comment:
There are two related statements:


- §2.1 JsonSchema says that, w.r.t. the `JsonSchema` class, "When dereferencing the `id` property associated with the `JsonSchema` type value the result is a valid JSON Schema document according to its specification version.".
- the statement above that I have just highlighted

There are two alternative interpretations of these two statements when put together.

1. The `jsonSchema` property could be seen _as an alternative_ to the dereferencing process for a `JsonSchema` class instance to get to a real Json Schema. In other words, the usage of the `jsonSchema` property is not related to the notion of `JsonSchemaCredential`, and it would be perfectly fine to use a `jsonSchema` property in the example 1, by simply "folding in" Example 2 into the `credentialSchema` construct.

2. The usage of `jsonSchema` is ***restricted to the situation where the `JsonSchema` class instance is the object of the `credentialSubject` property within a `JsonSchemaCredential` instance***. 

I suspect that the intention of the spec is alternative (2). If so, this statement should be very explicitly added to the definition of the property in §2.1 (I know that it is between the lines in §2.2, but not really explicitly). Unfortunately, expressing that in the RDFS ontology is not really possible (or would require some complex OWL ontology construct that I do not think we should use), which means one more reason for this restrictions to appear very explicitly.

Note that, personally, I would not have any problems with alternative (1), which is simpler. But it is not my decision.

Reply:
> Unfortunately, expressing that in the RDFS ontology is not really possible (or would require some complex OWL ontology construct that I do not think we should use), which means one more reason for this restrictions to appear very explicitly.

Actually, there would be one way of introducing this restriction into the vocabulary: define a _subclass_ of `JsonSchema` class, something like `EmbeddedJsonSchema`, define `jsonSchema`'s domain as being `EmbeddedJsonSchema`, and requiring that the object of a `credentialSchema` property, when used in conjunction with `JsonSchemaCredential`, would be of type `EmbeddedJsonSchema`. A bit more complex, but it works. Not sure it is worth it, though...

Reply:
@iherman please review the latest change, I've added a note with the text you've suggested in section 2.1